test/integration/web_console_access_test.go

```diff
@@ -17,7 +22,22 @@ import (
 	testserver "github.com/openshift/origin/test/util/server"
 )
 
-func tryAccessURL(t *testing.T, url string, expectedStatus int, expectedRedirectLocation string) *http.Response {
+func templateEscape(s string) (string, error) {
+	temp := new(bytes.Buffer)
+
+	t, err := template.New("foo").Parse(`{{define "T"}}{{.}}{{end}}`)
```

Comment:
I think https://github.com/sgallagher/origin/pull/1 is what we want to do for that test... if that looks good to you, squash it into your test commit and we can merge this